src/srcmak.ts

```diff
@@ -44,5 +44,12 @@ export async function srcmak(srcdir: string, options: Options = { }) {
       await fs.mkdirp(target); // make sure target directory exists
       await ncp(source, target, { clobber: false });
     }
+
+    if (options.csharp) {
+      const reldir = options.csharp.namespace;
+      const source = path.resolve(path.join(workdir, 'dist/dotnet/', reldir));
+      const target = path.join(options.csharp.outdir, reldir);
+      await fs.move(source, target, { overwrite: true });
```

Comment:
How does `dotnet` package dependencies work? Just wondering if we should do it similarly to java with:

```
await ncp(source, target, { clobber: false });
```

Reply:
I think it makes sense as is since each module generates a standalone project. Applications would then have project dependencies on the generated projects.

Reply:
Great!